src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Debug/LaunchSettingsJsonEncoding.cs

```diff
@@ -395,4 +395,26 @@ void ReadObject(Action<string> callback)
             }
         }
     }
+
+    /// <summary>
+    /// An implementation of <see cref="JsonTextReader"/> that skips any comments found
+    /// in the JSON data.
+    /// </summary>
+    private sealed class CommentSkippingJsonTextReader : JsonTextReader
+    {
+        public CommentSkippingJsonTextReader(TextReader reader) : base(reader)
+        {
+        }
+
+        public override bool Read()
```

Comment:
Does this mean that you won't roundtrip them?

Reply:
Yes but they don't round trip in 17.1 either.